src/test/java/ibm/jceplus/junit/base/BaseTestMD5.java

```diff
@@ -126,7 +126,7 @@ public void testMD5() throws Exception {
         md.update(array3);
         byte[] result = md.digest();
 
-        assertTrue("Digest did not match expected", Arrays.equals(result, md5result));
+        assertTrue(Arrays.equals(result, md5result), "Digest did not match expected");
```

Comment:
Should we use an `assertArraysEqual()` here and in all other instances in this test?